src/Sema.zig

```diff
@@ -9733,6 +9730,9 @@ fn funcCommon(
             .generic_owner = sema.generic_owner,
             .comptime_args = sema.comptime_args,
         });
+        const func_decl = mod.declPtr(ip.indexToKey(func_index).func.owner_decl);
+        func_decl.fqn =
+            try ip.namespacePtr(func_decl.src_namespace).internFullyQualifiedName(pt, func_decl.name);
```

Comment:
Wait, what's going on here? Shouldn't this fqn have already been inited in this exact way by `Zcu.PerThread.ScanDeclIter.scanDecl` or whatever?

Reply:
In `InternPool.finishFuncInstance`, `func.owner_decl` is assigned to a fresh decl with no fqn.